tests/components/glances/test_config_flow.py

```diff
@@ -54,6 +57,21 @@ async def test_form_cannot_connect(hass: HomeAssistant, mock_api: MagicMock) ->
     assert result["errors"] == {"base": "cannot_connect"}
 
 
+async def test_form_invalid_auth(hass: HomeAssistant, mock_api: MagicMock) -> None:
```

Comment:
To reduce the amount of double tests, you can use `@pytest.mark.parametrize` to quickly test some errors. An example:
https://github.com/home-assistant/core/blob/d76c5ed351941038898e2c692d59108eda70b282/tests/components/lastfm/test_config_flow.py#L58